You are a researcher analyzing the effectiveness of a new medication. The company claims that the medication reduces blood pressure by an average of 15 mmHg. You want to test if the medication actually reduces blood pressure by less than this amount.

**Given:**

*   Population Mean (
$\mu_0$
 ): 15 mmHg

*   Population Standard Deviation (
$\sigma_0$
 ): 4 mmHg

*   Sample Size (
𝑛
): 30 patients

*   Significance Level (
𝛼
): **0.01**

**Null Hypothesis:**

*   $H_0$
 :
𝜇
≥
15 mmHg (The medication reduces blood pressure by at least 15 mmHg)

*   $H_a$
​
 :
𝜇
<
15 mmHg (The medication reduces blood pressure by less than 15 mmHg)

In [1]:
import pandas as pd
df = pd.read_csv('/content/blood_pressure_reduction_data.csv')
df

Blood Pressure Reduction (mmHg)
0                         15.986857
1                         13.446943
2                         16.590754
3                         20.092119
4                         13.063387
5                         13.063452
6                         20.316851
7                         17.069739
8                         12.122102
9                         16.170240
10                        12.146329
11                        12.137081
12                        14.967849
13                         6.346879
14                         7.100329
15                        11.750850
16                         9.948676
17                        15.256989
18                        10.367904
19                         8.350785
20                        19.862595
21                        13.096895
22                        14.270113
23                         8.301007
24                        11.822469
25                        14.443690
26                         9.396026
27                        15.502792
28                        11.597445
29                        12.833225

In [4]:
# getting all the blood pressure reduction values into a variable:

bloodpressure_values = df['Blood Pressure Reduction (mmHg)'].values
bpv = bloodpressure_values

# calculating the mean of all bpv values:
mean_bpv = bpv.mean()
print(f'\nThe sample mean (x̄) of our 30 patients is:\n\t{round(mean_bpv, 3)}\n')


The sample mean (x̄) of our 30 patients is:
	13.247



The sample mean of the blood pressure reduction (x̄) is: **13.247**

We know need to calculate our sample standard deviation:

$\sigma_S$ = $\frac{\sigma}{\sqrt{n}}$


In [8]:
#calculating our sample standard deviation (σ sub S):

import numpy as np

sigma = 4  # population sd
n = 30  # sample size

ssd = sigma / np.sqrt(n)
print(f'The sample standard deviation (σ sub S) is:\n\t{round(ssd, 3)}')

The sample standard deviation (σ sub S) is:
	0.73


Our sample standard deviation is:

$\sigma_S$ = 0.73

We know need to get our p value from the z score table.
Before that we need to calculate the z score:

z score = $\frac{\bar{x} - \mu}{\sigma_S}$


In [11]:
from scipy.stats import norm

z_score = (mean_bpv - 15) / ssd
print(f'The z score is:\n\t{round(z_score, 3)}')
p_value = norm.cdf(z_score)

print(f'With this z score, our p value is:\n\t{round(p_value, 3)}')

The z score is:
	-2.4
With this z score, our p value is:
	0.008


Now we have our p value, we need to check this:

$p < \alpha$

And so we have:

0.008 < 0.01

**This is true.**

# CONCLUSIONS:

As a conclusion, it is unlikely to have x̄ = 13.247 **GIVEN** $\mu$ and $\sigma$.

Therefore we reject the null hypothesis in favor of the alternative hypothesis.

Indeed, based on the data of 30 patients, we do not have enough evidence to prove that the blood pressure reduction in a patient is equal to or greater than 15 mmHg.

Given the difference between a p value of 0.008 and the significance level of 0.01, we can say with enough confidence that the medication **does not reduce blood pressure by at least 15 mmHg.**

## Recommendations:  

*   a. Bigger sample size:  
Our conclusions could be refined with a much larger sample size. I suggest to conduct a study on at least 120 patients, split in 2 groups: control and experiment groups.  

*   b. Add variables to the study:  
Variables should also be tracked like physical activity level, diet, additional treatment or medications, etc. They will help identify other correlations and potentially refine our conclusions.
